.circleci/config.yml

```diff
@@ -515,19 +508,7 @@ jobs:
             --provision \
             -t TEST_TOKEN \
             --clusterConfig ~/repo/test/selenium/lib/compose/single \
-            --image rundeck/rundeck \
```

Comment:
this is not redundant in the context of the ci tests, it needs to be `rundeck/rundeck` or `rundeck/rundeck:latest` because that is the tag built locally. if you use SNAPSHOT it will pull from dockerhub, not what we want

Reply:
Argh, that succeeded for me locally :( OK, I'll add it back